bin/run_dsm.py

```diff
@@ -9,13 +9,12 @@
 
 async def go_to_enabled(csc_domain, options):
     remote = salobj.Remote(domain=csc_domain, name="DSM", index=options.index)
+    await remote.start_task
     await remote.evt_heartbeat.next(flush=False, timeout=120)
 
-    commands = ['start', 'enable']
+    commands = ["start", "enable"]
     for command in commands:
-        cmd = getattr(remote, f'cmd_{command}')
-        if command == 'start':
-            cmd.set(settingsToApply="default")
+        cmd = getattr(remote, f"cmd_{command}")
```

Comment:
Please consider using `await salobj.set_summary_state(...)` to replace lines 15-18. It's a single call to go from any initial state to the desired state.

Reply:
Does the call transition through all intervening states? I can't remember if it does.

Reply:
Yes.

Reply:
Done.